### PR TITLE
add verification script

### PR DESCRIPTION
**Pull request by asusevski**
Resolve gh issue Verification Script #4

### CODE REVIEW

**Review by SuryaKrishna02 — CHANGES_REQUESTED** [review]
Could you please update the `requirements.txt` file with all the libraries used in the verification script?

**Comment by asusevski** [comment]
Resolved all issues I believe, please review :)

File: translation/verification.py [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    llm_judge_appropriate_wording = []
+    llm_judge_fully_consistent = []
+
+    for reference, prediction in zip(references, predictions):
+        if do_repeated_tokens:
+            repeated = check_repeated_tokens(prediction, reference, tokenizer)
+            repeated_tokens.append(repeated)
+        if do_back_translate_chrf:
+            # for now all backtranslations are into english, add lang id later
+            response = translate(co, prediction, "en")
```
Comment (SuryaKrishna02) [inline translation/verification.py:146]:
```suggestion
            response = translate(co, prediction, "English")
```
Giving language code in the prompt might confuse the model. So, I think spelled out language is better.

**Review by SuryaKrishna02 — CHANGES_REQUESTED** [review]

**Comment by asusevski** [comment]
I added the requested changes + added to constants.py a map from iso lang code to a verbose spelling of the language for llm consumption 👍

**Review by SuryaKrishna02 — APPROVED** [review]
LGTM! Thanks @asusevski for your great work!!

**Pull request merged**